README.md

```diff
@@ -1,33 +1,47 @@
-# Databox
+# DataBox
+The Databox platform is an open-source personal networked device, augmented by cloud-hosted services, that collates, curates, and mediates access to an individual’s personal data by verified and audited third party applications and services. The Databox will form the heart of an individual’s personal data processing ecosystem, providing a platform for managing secure access to data and enabling authorised third parties to provide the owner with authenticated services, including services that may be accessed while roaming outside the home environment. Databox project is led by Dr. Hamed Haddadi (Imperial College) in collaboration with Dr. Richard Mortier (University of Cambridge) and Professors Derek McAuley, Tom Rodden, Chris Greenhalgh, and Andy Crabtree (University of Nottingham) and funded by EPSRC. See http://www.databoxproject.uk/ for more information.
 
-The Databox platform. See http://www.databoxproject.uk/ for more information
+## Getting Started
 
-## Installation
+These instructions will get you a copy of the project up and running on your local machine. For development and testing purposes, see Development section.
 
-All Databox components, including the container manager, run in Docker
-containers. So,
-first [install and run Docker](https://www.docker.com/products/docker), and
-then
-[install `docker-compose`](https://docs.docker.com/compose/install/#install-compose).
+### Prerequisites
 
-## Operation
+1) Requires Docker. Read [here](https://docs.docker.com/engine/installation/) for docker installation.
+2) Once docker is installed and running, install  docker-compose. Read [here](https://docs.docker.com/compose/install/) for installation. 
+3) Requires Git (if it is not already on your machine). Read [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) for git installation.
```

Comment:
Is this installation for development, or installation to try it out?
Not clear to me why we would require `git` unless doing development...? Isn't this just a pile of containers to run?

Reply:
Yes, git just pull all scripts and compose files. Not the components source files.